Phonebook/local-values.yml

```diff
@@ -19,14 +19,33 @@ source:
 
 traefik:
   enabled: true
-  startupArguments:
-    - ''
+  additional:
+    checkNewVersion: false
+    sendAnonymousUsage: false
```

Comment:
is `local-values.yml` for the preview? Can we activate in the preview the anonymous usage report to help the traffic developers?  I don't see why we must disable this in the preview.

Reply:
Nope, it is for local development only.